emacs.d/b-lsp.el

```diff
@@ -0,0 +1,19 @@
+(provide 'b-lsp)
+
+(require 'package)
+(setq package-user-dir (concat (file-name-directory load-file-name) "melpa"))
```

Comment:
Should this be in bivio-init.el?

Reply:
Up to you. My intention with putting everything in `b-lsp.el` was that if you decide to remove lsp in the future you can just delete this file. If you spread out the configuration then you'll have to find each bit and remove it. In my own config I've been slowly ripping things out of my main init file and into files like this. But, its just personal preference.

Reply:
Makes sense leave as is.